airflow/dags/kernel_documents.py

```diff
@@ -143,72 +143,49 @@ def list_documents(**kwargs):
     logging.debug("list_documents OUT")
 
 
-def read_xmls(**kwargs):
+def documents_to_delete(sps_package, sps_xml_files):
+    docs_to_delete = []
+    with ZipFile(sps_package) as zf:
+        for sps_xml_file in sps_xml_files:
+            logging.info("Reading XML file: %s" % sps_xml_file)
+            xml_content = zf.read(sps_xml_file)
+            if len(xml_content) > 0:
+                xml_file = etree.XML(xml_content)
+                scielo_id = xml_file.find(".//article-id[@specific-use='scielo']")
+                if scielo_id is not None:
+                    logging.info("  SciELO ID: %s" % scielo_id.text)
```

Comment:
Hm eu acho que essa mensagem de log não acrescenta nada em termos de contexto ou informação. Talvez ela não devesse existir. 

Penso que seria importante emitir uma mensagem quando `scielo_id is None` e `bool(scielo_id.text) == True`, pq quebraria uma invariante do sistema. Algo como `logging.info('cannot read scielo id from "%s": missing element in XML', sps_xml_file)`